api/tests/test_user.py

```diff
@@ -97,20 +97,6 @@ def test_user_email_is_required(self):
                 cohort=20, slack_handle='@test_user1',
                 password='devpassword')
 
-    def test_user_cohort_is_required(self):
```

Comment:
@egichuri According to [this](https://www.pivotaltracker.com/story/show/162193628) story, I think we need to maintain this test

Reply:
if so we can have a default cohort.. something like 00

Reply:
@MadgeKinyanjui